content/millennium/r4/general-clinical/allergy-intolerance.md

```diff
@@ -172,3 +167,68 @@ The `ETag` response header indicates the current `If-Match` version to use on a
 ### Errors
 
 The common [errors] and [OperationOutcomes] may be returned.
+
+## Update
+
+Update an existing allergy.
+
+    PUT /AllergyIntolerance/:id
+
+_Implementation Notes_
+
+* Any field which is missing will be interpreted as nulling out or removing data from the resource. See [FHIR<sup>®</sup> Update] for additional details about update operations.
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', 'Content-Type': 'application/fhir+json', 'If-Match': 'W/"&lt;Current version of the AllergyIntolerance resource>"'} %>
+
+### Body fields
+
+Notes:
+
+* The following fields cannot be modified: `code`, `patient`, `reaction.manifestation`, or an existing `note`.
+
+<%= definition_table(:allergy_intolerance, :update, :r4) %>
+
+### Example
+
+#### Request
+
+    PUT https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/AllergyIntolerance/6167733
+
+#### Body
+
+<%= json(:r4_allergy_intolerance_update) %>
+
+#### Response
+
+<%= headers status: 200 %>
+<pre class="terminal">
+    Cache-Control: no-cache
+    Content-Length: 0
+    Content-Type: application/json
+    Date: Thu, 05 Dec 2019 17:21:08 GMT
+    Etag: W/"8167765"
+    Last-Modified: Thu, 05 Dec 2019 17:21:08 GMT
+    Server-Response-Time: 743.187771
+    Status: 200 OK
+    X-Request-Id: a53f6469ff29031e9197b40f526e9ca6
+    X-Runtime: 0.743187
+    Vary: Origin
+</pre>
+
+The `ETag` response header indicates the current `If-Match` version to use on subsequent updates.
+
```

Comment:
Do we want to do the disclaimer here
```
<%= disclaimer %>
```

Reply:
Yeah we do, added: https://github.com/cerner/fhir.cerner.com/pull/327/commits/31f33cdc9cac89a6e5fc99c1a6d77e2683f8d674